### DISCUSSION TITLE
Allow transparent location arrow

### DISCUSSION THREAD

**Discussion by printmaps** [devemux86/cruiser#652] [Q&A] [answered]
Allow an alternative map cursor ( e. g. with transparency in order to hide less of the underlaying map):

Cruiser:
![Map-Cursor-Cruiser](https://github.com/user-attachments/assets/7178721b-b006-4586-aabb-4bbfcfe246d9)

Locus Map:
![Map-Cursor-Locus](https://github.com/user-attachments/assets/6d262a40-8036-4dcb-8a8a-2a2baa7d7ee8)

**Comment by DolomitesTom** [comment]
Why not, could be used.

**Comment by devemux86** [comment, marked as answer]
You can set the location at arrow tip to have more visibility in front of the arrow.
(Settings | Location)

<img src="https://github.com/user-attachments/assets/536fa281-84cc-4c84-a15b-ece100440254" width="50%"/><img src="https://github.com/user-attachments/assets/cc9a663e-e47c-4a00-9c22-584ec44e441e" width="50%"/>
